TUTORIAL.md

```diff
@@ -127,7 +128,9 @@ Now, start it up...
 
 ### Run the Spring Boot App
 
-To make your Spring Boot app aware of your Okta settings, you need to set a few environment variables. You can also use [other options](https://docs.stormpath.com/java/servlet-plugin/config.html) for setting these variables.
+To make your Spring Boot app aware of your Okta settings, you need to set a few environment variables. You can also use [other options](https://docs.stormpath.com/java/servlet-plugin/config.html) for setting these variables. 
+
+// you said this already, look up there ^
```

Comment:
I believe you. Please feel free to delete paragraphs and make direct edits in your PR. I'll question/comment if I disagree.